dired-rifle.el

```diff
@@ -42,11 +42,20 @@
           (const :tag "Default" nil)
           (string :tag "Custom")))
 
+(defcustom dired-rifle-use-marked-files nil
+  "If non-nil, dired-rifle will work on marked files instead of the
+file under point. But 'file under point' will be used as fallback, if
+there are no marked files"
+  :type '(choice
+          (const :tag "single-file" nil)
+          (const :tag "marked-files" t))
+  )
```

Comment:
I must have missed it before, let's put this parenthesis on the previous line. This is the style most commonly used in Lisps.